samples/sensor/bmm150/README.rst

```diff
@@ -0,0 +1,38 @@
+.. _bmm150:
+
+BMM150 GeoMagnetic Sensor
+#########################
+
+Overview
+********
+
+This sample application periodically reads magnetometer (X, Y, Z) data from
+the first available device that implements SENSOR_CHAN_MAGN_* (predefined array
+of device names). This sample checks the sensor in polling mode (without
+interrupt trigger).
+
+Building and Running
+********************
+
+This sample application uses an BMM150 sensor connected to an Arduino 101 board via I2C.
```

Comment:
Can you provide a reference to how to connect the sensor, and for that matter, what other hardware is needed to flash the Arduino 101 in the steps below.  Is a USB cable sufficient?

Reply:
USB and I2C is enough for connection. and for extra pins I added datasheet reference